### DISCUSSION TITLE
"New Contributors" section

### DISCUSSION THREAD

**Discussion by lmittmann** [goreleaser/goreleaser#3928] [Q&A] [answered]
Many releases of repos using GoReleaser have a section "New Contributors". Can this be generated by GoReleaser? 

Example release: https://github.com/goreleaser/goreleaser-action/releases/tag/v4.1.1

**Comment by caarlos0** [comment, marked as answer]
yes:

```yaml
changelog:
  use: github-native
```

docs: https://goreleaser.com/customization/changelog/

**Reply by lmittmann** [reply]
Is there any way to customize this section using e.g. a template?

**Reply by caarlos0** [reply]
no... you can add a header and/or a footer though. 

All the options are available in the docs link above.
